R/wwinference.R

```diff
@@ -62,10 +62,7 @@
 #'
 wwinference <- function(ww_data,
                         count_data,
-                        model_spec = wwinference::get_model_spec(
-                          forecast_date =
-                            "2023-12-06"
-                        ),
+                        model_spec = wwinference::get_model_spec(),
                         mcmc_options = wwinference::get_mcmc_options(),
                         generate_initial_values = TRUE,
                         compiled_model = wwinference::compile_model()) {
```

Comment:
Since these functions used for defaults are defined within the same package as the `wwinference()` function, they shouldn't need the `wwinference` prefix.